packages/toolkit/src/view/model/ModelsTable.tsx

```diff
@@ -61,7 +62,9 @@ export const ModelsTable = (props: ModelsTableProps) => {
               secondaryLink={null}
               secondaryText={label}
               iconElement={getIcon(
-                `w-4 h-4 ${["TASK_CHAT"].includes(row.original.task || "") ? "stroke-semantic-fg-primary [&>*]:!stroke-semantic-fg-primary" : "[&>*]:!fill-semantic-fg-primary"}`,
+                cn(
+                  `w-4 h-4 ${["TASK_CHAT", "TASK_CUSTOM"].includes(row.original.task || "") ? "stroke-semantic-fg-primary [&>*]:!stroke-semantic-fg-primary" : "[&>*]:!fill-semantic-fg-primary"}`,
+                ),
```

Comment:
```suggestion
                cn(
                  "w-4 h-4",
                  ["TASK_CHAT", "TASK_CUSTOM"].includes(row.original.task || "") ? "stroke-semantic-fg-primary [&>*]:!stroke-semantic-fg-primary" : "[&>*]:!fill-semantic-fg-primary"
                ),
```

Reply:
Makes sense. Sorry, didn't really think about code style because of how minor the changes are.